site/content/en/blog/releases/1.30.0.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "1.30.0 - GKE Autopilot and Multiple Kubernetes version support"
+linktitle: "1.30.0"
+date: "2023-02-28"
+---
+
+This is the 1.30.0 release of Agones.
+
+This first stable release without a release candidate brings us support for multiple Kubernetes versions (no more being tied to a single version!) along with GKE Autopilot support, as well as the usual slew of bug fixes and general improvements.
```

Comment:
I'm not sure I'd call this out the fact that we didn't do an RC as the _first_ thing in the release notes. What about:

> This release brings us support for multiple Kubernetes versions (no more being tied to a single version!), support for GKE Autopilot, and a large collection of bug fixes and general improvements. This is also the first release following our new release process whereby we do not publish a release candidate prior to the release.